pkg/jsonschemautil/testdata/instance.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Why does this file exist? It seems to be unused by the test.

Reply:
It was the original upstream test, I converted it to yaml